docs/installation/helm.md

```diff
@@ -1,75 +1,172 @@
-# Helm 
+# Helm
 
+This comprehensive guide will help you install SigLens on your local Minikube Kubernetes cluster using Helm. Helm simplifies the deployment of Kubernetes applications, making it a breeze to set up SigLens.
```

Comment:
we should have an easy first way of installing siglens via helm, which is just running those couple of cmds that we have. 
And then below that, mention "Install using minikube" and all these details about minikube and stuff will follow there